src/script/vco-ie/main_warhammer/wh_main_brt_bordeleaux/missions.lua

```diff
@@ -116,10 +116,10 @@ local missions = {
 				objective
 				{
 					override_text mission_text_text_vco_brt_bdx_3_home_sweet_home;
-					type CONTROL_N_PROVINCES_INCLUDING;
+					type CONTROL_N_REGIONS_INCLUDING;
 					total 1;
 
-					province wh3_main_combi_province_river_brienne;
```

Comment:
@OhManTFE 

Why did you introduce this change? Is it defined as a region or as a province in the docs?

Reply:
Docs says Bordeleaux. Mission text says his ancestral castle, which is Bordeleaux. "River Brienne" is just confusing and not loreful. I changed it to what it was supposed to be in the first place.